src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/OperationExtensions.cs

```diff
@@ -185,6 +208,26 @@ public static ValueUsageInfo GetValueUsageInfo(this IOperation operation)
             return ValueUsageInfo.Read;
         }
 
+        public static IMethodSymbol TryGetContainingAnonymousFunctionOrLocalFunction(this IOperation operation)
```

Comment:
NRT-ise this  file?

Reply:
I updated it locally, but will submit that change separately to master branch.